tests/integrations/config/dynamic/mod.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+mod raftstore;
+mod split_check;
```

Comment:
We can also move gc_worker relative tests here.

Reply:
Sure!